apps/meteor/client/views/admin/moderation/hooks/useResetAvatarAction.tsx

```diff
@@ -25,8 +25,11 @@ const useResetAvatarAction = (userId: string): GenericMenuItemProps => {
 
 	const onResetAvatar = async () => {
 		setModal();
-		handleResetAvatar.mutateAsync({ userId });
+		await handleResetAvatar.mutateAsync({ userId });
 		queryClient.invalidateQueries({ queryKey: ['moderation.reports'] });
+		queryClient.invalidateQueries({ queryKey: ['moderation.usersReport', { userId }] });
```

Comment:
what about changing yours querykey to `['moderation', ...]` 
you can invalidate all of them just using `['moderation']`
```suggestion
		queryClient.invalidateQueries({ queryKey: ['moderation', 'usersReport', { userId }] });
```

Reply:
That's a good idea. Thank you! Renamed the keys.